BraveWallet/Crypto/Stores/SendTokenStore.swift

```diff
@@ -150,8 +152,14 @@ public class SendTokenStore: ObservableObject {
     let weiFormatter = WeiFormatter(decimalFormatStyle: .decimals(precision: 18))
     guard let token = selectedSendToken, let weiHexString = weiFormatter.weiString(from: amount, radix: .hex, decimals: 18) else { return }
     
+    isMakingTx = true
     rpcController.network { [weak self] network in
-      guard let self = self else { return }
+      guard let self = self else {
+        self?.isMakingTx = false
+        return
+      }
```

Comment:
These are not needed, if this guard fails then self is nil and won't execute any of this